src/current/_includes/releases/cloud/2023-06-05.md

```diff
@@ -13,7 +13,7 @@
 
 <h3> Bug fixes </h3>
 
-- The `status` returned by the [`logexport` Cloud API endpoint](../cockroachcloud/export-logs.html#the-logexport-endpoint) is now determined by the readiness of the underlying logging resources, in addition to the latest log export update job state. Before this change, it was possible for `GET` requests to the `logexport` to report a log export status that conflicted with the latest log export update job state or with the state of the logging infrastructure.
-- Fixed a bug where concurrent [restores](../cockroachcloud/use-managed-service-backups.html#restore-a-cluster) could run on the same destination cluster.
+- The `status` returned by the [`logexport` Cloud API endpoint](../cockroachcloud/export-logs.html#the-logexport-endpoint) is now determined by the state of both the latest log export's job state and the readiness of the underlying logging resources. Before this change, a `GET` request to the `logexport` endpoint could report an outdated log export status that conflicted with the latest log export update job state or with the most recent state of the logging infrastructure.
+- Fixed a bug where concurrent [restores](../cockroachcloud/use-managed-service-backups.html#restore-a-cluster) could run on the same destination cluster and cause the destination cluster to become unusable.
 - Fixed a bug where the IOPS price preview shown when [creating](../cockroachcloud/create-your-cluster.html) or [editing a cluster](../cockroachcloud/cluster-management.html) was inaccurate.
-- The **Group** tab is now only shown to users who have access to it.
+- The **Group** tab is now only shown to users who have this feature enabled. Previously, an error page was shown to users who navigated to the **Group** tab without access.
```

Comment:
```suggestion
- The **Group** tab is now shown only to users who have this feature enabled. Previously, an error page was shown to users who navigated to the **Group** tab without enabling the feature.
```
Moving "only" is an optional nit and I can't really explain why it seems better to me.
Changed "without access" based on your comment about what was fixed. Is it more accurate?